diagnostic.md

```diff
@@ -49,8 +53,11 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
-```
+@function tint($color, $percentage) {
+  @return mix(#fff, $color, $percentage);
```

Comment:
This will probably work, but SASS may complain - better to use the `$pure-white` trick we discussed in class. No points off, though.